examples/lm1b/train.py

```diff
@@ -181,7 +184,7 @@ def compute_metrics(logits, labels, weights, label_smoothing=0.0):
 # -----------------------------------------------------------------------------
 
 
-def train_step(optimizer,
+def train_step(opt_state,
```

Comment:
We usually refer to `TrainingState` as `state`, not as `opt_state`.

Reply:
Done.